README-tg.md

```diff
@@ -0,0 +1,290 @@
+<p align="center">
+    <img alt="oss image" src="./imgs/zoss-logo.svg" height="200px" width="200px">
+</p>
+
+<p align="center">
+  <a href="https://twitter.com/intent/tweet?text=How%20to%20promote%20your%20open-source%20projects%20@ZenikaOSS&url=https://github.com/zenika-open-source/open-source-promotion-cheat-sheet&hashtags=OpenSource,CheatSheet">
+    <img alt="tweet" src="https://img.shields.io/twitter/url/https/twitter?label=Share%20on%20twitter&style=social" target="_blank" />
+  </a>
+</p>
+
+<h1 align="center">Роҳнамои моъарефи ва тосъи лоиҳаҳои матни боз</h1>
+
+
+> Ин маҳтави ба шумо комак миконад то барноме "матни боз" ход ро бехтар ва сарих тар моъарефи конид. 
+
+Лист соӣр забонҳои муҷуд:
+
+- [English](./README.md)
+- [Français](./README-fr.md)
+- [Deutsch](./README-de.md)
+- [Тоҷикӣ](./README-tg.md)
+
+A language is missing? Want to improve this cheat sheet? [Contributions are welcome](./CONTRIBUTING.md)!
```

Comment:
I think we should translate this sentence as well.
I know it hasn't been translated in `README-fa.md`, but I think it's probably a mistake or an oversight.